cmd/list/rhRegion/cmd.go

```diff
@@ -0,0 +1,72 @@
+package rhRegion
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/config"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var (
+	args struct {
+		discoveryURL string
+	}
+)
+
+var Cmd = NewRhRegionCommand()
+
+func NewRhRegionCommand() *cobra.Command {
+	Cmd := &cobra.Command{
+		Use:   "rh-regions",
+		Short: "List available OCM regions",
+		Long:  "The command lists available OpenShift Cluster Manager regions.",
+		Example: `  # List all supported OCM regions 
+ocm list rh-regions`,
+		Run:    run,
+		Hidden: true,
+		Args:   cobra.NoArgs,
+	}
+
+	flags := Cmd.Flags()
+	flags.StringVar(
+		&args.discoveryURL,
+		"discovery-url",
+		"",
+		"URL of the OCM API gateway. If not provided, will reuse the URL from the configuration "+
+			"file or "+sdk.DefaultURL+" as a last resort. The value should be a complete URL "+
+			"or a valid URL alias: "+strings.Join(ocm.ValidOCMUrlAliases(), ", "),
+	)
+	return Cmd
+}
+
+func run(cmd *cobra.Command, argv []string) {
```

Comment:
@tirthct Could we follow the same pattern we used for `get`/`set` config where we have a function that takes in parameters and returns the `error` with a single `os.Exit` call. That makes this function testable. Bonus points for that as well 👍

Reply:
Added this in the separate method. Also added a really small test case to confirm error doesn't exist. Since we are using tabwriter, could not check the output